tests/mod.rs

```diff
@@ -0,0 +1,75 @@
+use assert_cmd::{assert::Assert, Command};
+
+const MOUNTPOINT: &str = "/tmp/rmnt";
+
+fn create_mountpoint_dir() {
+    Command::new("/bin/mkdir").args(["-p", MOUNTPOINT]).output().unwrap();
+}
+
+fn remove_mountpoint_dir() {
+    Command::new("/bin/rm").args(["-Rf", MOUNTPOINT]).output().unwrap();
+}
+
+fn run_rfs_detached() -> Assert {
+    Command::new("echo").args(["user_allow_other", ">>", "/etc/fuse.conf"]).output().unwrap();
+    let mut cmd = Command::cargo_bin("rfs").unwrap();
+    let assert_mount = cmd
+        .args([
+            "-d",
+            "--log",
+            "/tmp/rfs.logs",
+            "--meta",
+            "./tests/threefoldtech-funk-latest.flist",
+            MOUNTPOINT,
+        ])
+        .assert();
+
+    let logs = Command::new("cat").args(["/tmp/rfs.logs"]).output();
```

Comment:
Seriously ? Is this more efficient than opening the file and reading it 
Look, What you do now , is that you fork, and exec a new process this process then opens the file reads it and prints it out to it's `stdout` you then pipe the stdout and collect ALL output into you memory then you have `logs`. Is this more efficient than `open(/tmp/rfs.logs)` then doing a read or even read all ??!!!

You need to learn how to program things !